bower.json

```diff
@@ -34,5 +35,8 @@
         "sinon": "http://sinonjs.org/releases/sinon-1.17.0.js",
         "angular-mocks": "1.4.4"
     },
-    "private": true
+    "private": true,
+    "resolutions": {
```

Comment:
isn't there an earlier version of bootstrap-timepicker that doesn't require 2.0? This is a third party plugin, not part of the main twitter bootstrap. I would try to specify a version `bootstrap-timepicker#version` that still works with 1.9.

Hmm, looking at the change logs, what version of timepicker was found in bower? Looks like `0.2.6` was an accidental release---hah. A `0.2.7` upgrade is recommended or bumping down to `0.2.5` https://github.com/jdewit/bootstrap-timepicker/releases